pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -191,6 +191,13 @@ func (o *MustGatherOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, arg
 	if len(o.DestDir) == 0 {
 		o.DestDir = fmt.Sprintf("must-gather.local.%06d", rand.Int63())
 	}
+	if strings.HasPrefix(o.DestDir, "~/") {
```

Comment:
This seems to be specific to Unix and does not work on Windows. Besides, nearly all commands accept full path and does not accept `~/`. I'd prefer leave this as is.